### DISCUSSION TITLE
Create Translations Repository

### DISCUSSION THREAD

**Discussion by hishamco** [oqtane/oqtane.framework#971] [General]
If @sbwalker doesn't mind, please add a new repo called `Oqtane.Translations` that hosts the supported translations at least it is a good thing to test our Oqtane localization in both site and admin section. Give me a permission to commit and merge PRs so my work can go somoothly and fast as much as I can

Furthermore I can work on a tool that facilitate extracting the localization resources, so no need to guess what are the available resources, that tool can hosted in my account and all the contributions are welcome. I'm think of a way using GitHub actions or something similar, so whever a change to the source happen it runs and extracts the new localizations resources if there is

@chlupac @ADefWebserver @mikecasas @jimspillane  your thoughts

**Comment by ADefWebserver** [comment]
@hishamco Sounds like a good idea to me :)

**Comment by sbwalker** [comment]
Sorry for the delay... I will add this today

**Reply by sbwalker** [reply]
Done!

**Comment by hishamco** [comment]
Thanks @sbwalker for creating the repo, could please enable discussion features there, so any discussion related to the resources could be there instead of adding it here

**Reply by sbwalker** [reply]
@hishamco Discussions are still a beta feature and cannot be enabled on a repo by its owner yet. I would suggest that in the short term just use the Issues capability for this and once Discussions are out of Beta I will be able to enable them on all of Oqtane's repositories.
